### PR TITLE
Make sure framework does not exit when there's a problem decrypting cookies

### PR DESCRIPTION
**Pull request by juliusza**
Reproduce with 

`setcookie('dummy[foo][bar]', 'value1' , time()+3600);`

the next request will fail, with msg "string expected"

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "laravel/framework",
+    "name": "juliusza/framework",
```
Comment (jmarcher) [inline composer.json]: Why is this here? It is not your framwork...

File: src/Illuminate/Cookie/Guard.php [hunk 2]
```diff
@@ -65,7 +65,7 @@ protected function decrypt(Request $request)
 			{
 				$request->cookies->set($key, $this->decryptCookie($c));
 			}
-			catch (DecryptException $e)
+			catch (\Throwable $e)
```
Comment (jmarcher) [inline src/Illuminate/Cookie/Guard.php]: This is too vague... DecryptException is fine

**Review by jmarcher — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (juliusza) [reply composer.json]: not suggesting to merge this

File: src/Illuminate/Cookie/Guard.php (hunk 2 above)
Comment (juliusza) [reply src/Illuminate/Cookie/Guard.php]: have you read the my comment? it's not fine because users are able to crash framework by setting cookies in a specific way. Otherwise this can be solved by checking if value is string here:

`	protected function decryptArray(array $cookie)
	{
		$decrypted = array();
		foreach ($cookie as $key => $value)
		{
                        // $value could be array here, check if string
			$decrypted[$key] = $this->encrypter->decrypt($value);
		}
		return $decrypted;
	}`

Comment (mfn) [reply src/Illuminate/Cookie/Guard.php]: But what is the exception and message you actually got in your case?
